filegenerator/file_generator.go

```diff
@@ -121,14 +121,28 @@ func isValidDataDir(dataDirName string) bool {
 	return matched
 }
 
+func checkEnvSet(env string) {
```

Comment:
Maybe convert this to `mustGetenv` or something similar and add a call to `os.Getenv` here so that this can be called in place of `os.Getenv`